ent.xml

```diff
@@ -0,0 +1,392 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be somewhere else, like in build_tools with osx_distribution.xml.

Also ideally it would have a more descriptive name - "ios_ent.xml" (or whatever "ent" stands for).